.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+## How to contribute
+
+Share your knowledge with the community by contributing to Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [devdocs](https://GitHub.com/magento/devdocs) GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new topics.
+
+Magento's tech writer team and [Maintainers](https://devdocs.magento.com/contributor-guide/devdocs-maintainers.html) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
+Working through the backlog takes time, though, so we appreciate your patience.
+
+## Rewards and thanks
+
+DevDocs works with the Community Engineering teams and projects.
+As you contribute PRs, you gain [Contribution Points](https://github.com/magento/magento2/wiki/Contribution-Rewards).
+
+If you write and contribute a full topic, we will put your name (or your company's name) at the top of the page right under the title, and link it to your blog or website!
```

Comment:
Top of what page? Maybe link?

Reply:
Editing, and removing a second instance of this.